expat/fuzz/.gitignore

```diff
@@ -1 +1,3 @@
 /xml_parse*
+!/xml_parsebuffer_fuzzer.c
+!/xml_parse_fuzzer.c
```

Comment:
@t-0 could you combine this to `!/xml_parse*.c` to better scale for when someone adds the next file?

Reply:
Of course! Done.

Reply:
@t-0 thank you!